### PR TITLE
Add Cervisia

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** Please ping the KDE team.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/sdk/cervisia/-/merge_requests/7 https://invent.kde.org/sdk/cervisia/-/merge_requests/8

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
Note: while this compiles, the use of KLauncher does not work. There might be permission problems.

**Comment by Mailaender** [comment]
bot, build org.kde.cervisia

File: org.kde.cervisia.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "command": "cervisia",
+  "rename-icon": "cervisia",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--talk-name=org.kde.klauncher5",
+    "--socket=system-bus"
```
Comment (hfiguiere) [inline org.kde.cervisia.json:15]: why system-bus? You should instead figure out what names are needed.

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by Mailaender** [comment]
bot, build org.kde.cervisia

File: org.kde.cervisia.json (hunk 1 above)
Comment (Mailaender) [reply org.kde.cervisia.json]: Actually `session-bus` would have been correct, but I figured it out by looking at https://invent.kde.org/packaging/snapcraft-kde-applications/-/blob/Neon/release/cervisia/snapcraft.yaml#L32

Comment (hfiguiere) [reply org.kde.cervisia.json]: you can log the bus session when trying the build to see what is needed.

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "skip-icons-check": true
```
Comment (barthalion) [inline flathub.json:2]: This is a no-go.

**Comment by barthalion** [comment]
cc @flathub/kde

**Comment by Mailaender** [comment]
bot, build org.kde.cervisia

**Comment by Mailaender** [comment]
bot, build org.kde.cervisia

File: org.kde.cervisia.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    {
+      "name": "cervisia",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_TESTING=OFF"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/release-service/22.12.3/src/cervisia-22.12.3.tar.xz",
```
Comment (tsdgeos) [inline org.kde.cervisia.json:45]: 23.04.3

Comment (tsdgeos) [reply org.kde.cervisia.json:45]: But besides that, the fact that the app doesn't start is obviously a bigger problem ^_^

**Comment by Mailaender** [comment]
GUI now shows:

> The name org.kde.klauncher5 was not provided by any .service files

Terminal shows:
> kf.init.klauncher: D-Bus communication problem!
kdeinit5: Communication error with launcher. Exiting!
kf.notifications: Audio notification requested, but sound file from notifyrc file was not found, aborting audio notification

I have no idea how to fix this.

**Comment by Mailaender** [comment]
> kdeinit5: preparing to launch '/usr/lib/x86_64-linux-gnu/libexec/kf5/klauncher'

seems to be the issue

**Pull request closed without merging**